tests/pyincore/analyses/electricpowerfacilityrestoration/test_electricpowerfacilityrestoration.py

```diff
@@ -0,0 +1,28 @@
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License v2.0 which accompanies this distribution,
+# and is available at https://www.mozilla.org/en-US/MPL/2.0/
+
+
+from pyincore import IncoreClient, MappingSet, RestorationCurveSet, RestorationService
+from pyincore.analyses.electricpowerfacilityrestoration import ElectricPowerFacilityRestoration
+import pyincore.globals as pyglobals
+from pyincore.models.mapping import Mapping
+
+
+def run_with_base_class():
+    client = IncoreClient(pyglobals.INCORE_API_DEV_URL)
+    wf_rest = ElectricPowerFacilityRestoration(client)
+    restorationsvc = RestorationService(client)
+    mapping_set = MappingSet(restorationsvc.get_mapping("61f302e6e3a03e465500b3eb"))  # new format of mapping
+    wf_rest.set_input_dataset('dfr3_mapping_set', mapping_set)
+    wf_rest.set_parameter("result_name", "wf_restoration.csv")
```

Comment:
result name shouldn't include .csv. Also could you again rename it to epf_restoration to avoid confusion